private-templates-service/client/src/Config.tsx

```diff
@@ -0,0 +1,7 @@
+import { } from "react"; // Typescripte requires every file to have an import
+
+export default {
+  appId: "4803f66a-136d-4155-a51e-6d98400d5506",
```

Comment:
I think app registration should be fixed or removed from this PR. What do you think?

Reply:
^ I agree - we should also start setting client ids as secret